### PR TITLE
feat: add load default layout button to dev panel

### PR DESCRIPTION
**Pull request by gazuntype**
Screenshot 2022-03-29 at 18 06 02

added a button to load the default layout file. 

i moved the default layout file into the plugin to ensure artists don't delete it/move it.

also, i can also set it to load the default layout on default but i'm not sure if artists would want that cos personally, it'll be annoying. we can move the placement of the button though.

### CODE REVIEW

**Comment by gazuntype** [comment]
this change means we'd have to remove the layout from the template btw.

**Comment by chornesays** [comment]
Ah. I looked at this yesterday and then got distracted

Rather than a button - I was thinking it should be in the dropdown right above "logout" -- basically anything that is VERY rarely used and is more about setting up your entire "Filta State" should probably go in the Filta dropdown menu

File: Util/LayoutUtility.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+using Type = System.Type;
+
+public static class LayoutUtility {
+
+	private static readonly MethodInfo LoadWindowLayout;
+
+	static LayoutUtility() {
+		Type windowLayout = Type.GetType("UnityEditor.WindowLayout,UnityEditor");
+		if (windowLayout != null) {
+			LoadWindowLayout = windowLayout.GetMethod("LoadWindowLayout",
```
Comment (chornesays) [inline Util/LayoutUtility.cs:14]: this is some magic right here

**Comment by gazuntype** [comment]
> -- basically anything that is VERY rarely used and is more about setting up your entire "Filta State" should probably go in the Filta dropdown menu

hm hm makes sense.

**Pull request merged**